pkg/obfuscate/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/DataDog/datadog-agent/pkg/obfuscate
+
+go 1.16
+
+require (
+	github.com/DataDog/datadog-go v4.8.3+incompatible
+	github.com/Microsoft/go-winio v0.5.1 // indirect
+	github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575
```

Comment:
The versions here should match the ones on the main `go.mod` file (so datadog-go needs to be `v4.8.2+incompatible` and seelog is a 2015 pseudoversion)